src/components/result-view-tab.js

```diff
@@ -171,11 +174,11 @@ export const ResultViewTab = ({
                 </Tabs>
                 {disabled && <AlertInvalidNode />}
             </div>
-            {tabIndex === 0 && !disabled && renderLoadFlowResult()}
-            {tabIndex === 1 && !disabled && renderSecurityAnalysisResult()}
-            {tabIndex === 2 && !disabled && renderShortCircuitAnalysisResult()}
-            {tabIndex === 3 && !disabled && renderSensitivityAnalysisResult()}
-            {tabIndex === 4 && !disabled && renderDynamicSimulationResult()}
+            {tabIndex === 0 && showsSubs && renderLoadFlowResult()}
```

Comment:
The showTabs boolean could be used at the Paper level and we would have about the same behavior.

A smarter behavior would be : 1) not destroying the results tab component when it is not visible, so that we can have a very quick display without download when we click back on the results tab. 
2) but not downloading the results when the results tab is not visible and we change the current node for example.